swagger_output.json

```
@@ -7,9 +7,17 @@
   },
   "host": "auth.akuya.tech",
   "servers": [
+    {
+      "url": "localhost:3000",
+      "description": "local server"
+    },
     {
       "url": "auth.akuya.tech",
-      "description": "Confirmed working well"
+      "description": "development sever"
+    },
+    {
+      "url": "staging.zuri.team",
```

add /api/auth to this
```suggestion
      "url": "staging.zuri.team/api/auth",
```